src/dots.jsx

```diff
@@ -25,8 +25,9 @@ export var Dots = React.createClass({
       slidesToScroll: this.props.slidesToScroll
     });
 
-    var dots = Array.apply(null, {length: dotCount}).map((x, i) => {
+    var dots = [];
 
+    for (var i=0; i<dotCount; i++) {
```

Comment:
@nosir Thanks for your comment. My knowledge about ie8 is next to nothing. @sorrycc Is es5-shim an acceptable solution in your case? I really don't want to keep any for loops in my code unless it is required seriously.